### PR TITLE
modify error code nightly cases

### PR DESCRIPTION
**Pull request by NicoYuan1986**
#23075

### CODE REVIEW

**Comment by yanliang567** [comment]
@NicoYuan1986 please help to resolve the conflicts first.

File: tests/python_client/testcases/test_collection.py [hunk 1]
```diff
@@ -2862,7 +2863,16 @@ def test_load_replica_partitions(self):
                            check_task=CheckTasks.check_query_results,
                            check_items={'exp_res': df_2.iloc[:1, :1].to_dict('records')})
 
-        error = {ct.err_code: 1, ct.err_msg: f"not loaded into memory"}
+        error = {ct.err_code: 65538, ct.err_msg: "failed to query: attempt #0: failed to search/query delegator "
```
Comment (yanliang567) [inline tests/python_client/testcases/test_collection.py:2866]: why we assert such a long and specific error msg? the partition id shall be changed next time

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_collection.py:2866]: I will modify the message and extract keywords.

**Pull request merged**